### ISSUE TITLE
Create the LinearRegression Estimator

### ISSUE THREAD

**Issue by dcrescim** [opensource9ja/scikit.js#7]
**Is your feature request related to a problem? Please describe.**
In order to match the Estimators in scikit-learn, we should make a LinearRegression estimator with the same API.

**Describe the solution you'd like**
There are couple of ways to do this. Scikit-Learn leans on linear algebra libraries from scipy. I'm honestly not sure if JS (either Node / Browser) has adapters for the same low-level BLAS and LAPACK libraries that scipy uses. 
In lieu of not having access to those libraries, the plan should be to just create a gradient descent solution using a TF model.

Once we know / can use low level libraries for solving a linear system of equations, we should trade out the SGD solution above in favor of that.

**Additional context**
There might be cases where we actual want to have both implementations (Linear Algebra solver, and SGD), because we could deploy different ones to different contexts, ie... Lin Alg solver on Node, and SGD solver on web. This would get rid of the need for shipping an entire solver to the client. But that's something that we would need to test later.
